src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  let marriedStatus = '';
+
+  if (isMarried) {
+    marriedStatus =
+      sex === 'm'
+        ? `${partnerName} is my wife`
+        : `${partnerName} is my husband`;
```

Comment:
In both cases the string is almost the same, only the word wife/husband is different, so you can set only it conditionally